docker/webDriverIOCodeceptjs/README.md

```diff
@@ -0,0 +1,61 @@
+# Codeceptjs Nightmare
```

Comment:
Thanks 😄 , 

Great, we can use that organization docker hub to hold the image.

I see, I mainly created the Dockerfiles this way to keep them as light as possible, and perhaps easier to maintain. However, I can see that If a WebDriverIO changes, then subsequent images must be rebuilt. Likewise, if a new version of CodeceptJS is released, several images will have to be built and pushed. 

I have merged all Dockerfiles into one. It comes with Node, Nightmare, WebDriverIO and now Protractor. Additionally, I have updated the `README.md` to include running and building instructions.

_Note: I've kept the `npm install ...` commands separate to allow docker to cache them if there are no changes to them. For example, If there is a new version of CodeceptJS the build will not have to reinstall WebDriverIO or Nightmare or Protractor etc, it can just used the cached layer. Overall, this should speed up image build times._